js/components/KeyboardShortcutHUD.react.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * KeyboardShortcutHUD
+ */
+import React, { Component } from 'react';
+import Combokeys from 'combokeys'
+
+class KeyboardShortcutHUD extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { show: false };
+  }
+
+  componentDidMount() {
+    let combokeys = new Combokeys(document.documentElement);
+    combokeys.bind('?', () => { this.setState({ show: !this.state.show }); });
+  }
+
+  componentWillUnmount() {
+    let combokeys = new Combokeys(document.documentElement);
+    combokeys.unbind('?');
+  }
+
+  render() {
+    let className = this.state.show ? 'keyboard-hud' : 'keyboard-hud hidden';
+    return (
+      <div className={ className }>
+        <ul>
+          <li><span className="key">?</span> Toggle HUD</li>
+          <li><span className="key">&larr;</span> Previous Passage</li>
+          <li><span className="key">&rarr;</span> Next Passage</li>
+          <li><span className="key">1</span> Read state (Know)</li>
```

Comment:
Similarly, "state" language may be confusing to users without a background in programming or UI design.